go/engine/profile_edit.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package engine
+
+import (
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+type ProfileEdit struct {
+	libkb.Contextified
+	arg keybase1.ProfileEditArg
+}
+
+func NewProfileEdit(g *libkb.GlobalContext, arg keybase1.ProfileEditArg) *ProfileEdit {
+	return &ProfileEdit{
+		Contextified: libkb.NewContextified(g),
+		arg:          arg,
+	}
+}
+
+func (e *ProfileEdit) Run(ctx *Context) (err error) {
+	defer e.G().CTrace(ctx.NetContext, "ProfileEdit#Run", func() error { return err })()
+	_, err = e.G().API.Post(libkb.APIArg{
+		Endpoint:    "profile-edit",
+		NeedSession: true,
+		Args: libkb.HTTPArgs{
+			"bio":      libkb.S{Val: e.arg.Bio},
+			"fullname": libkb.S{Val: e.arg.FullName},
```

Comment:
Is there a bug here?  The profile-edit API endpoint takes `full_name`, not `fullname`.  And if so, how come your tests pass?